src/controllers/auth/login.ts

```diff
@@ -0,0 +1,21 @@
+import jwt from 'jsonwebtoken'
+import { usuarios } from '../../fakedata/usuarios.json'
+import { constants } from '../../config/constants'
+
+
+import { Request, Response} from "express";
+export default (req: Request, res: Response) => {
+    const credentials = req.body;
+    const match = usuarios.find(user => {
+        if (user.login === credentials.login && user.password === credentials.password){
+            return user;
+        }
+    });
+
+    if (!match) {
+        return res.sendStatus(404);
+    }
+    const { id, login } = match
```

Comment:
siempre deja un espacio antes y despues de un bloque IF, FOR, MAP, FIND, FILTER, REDUCE, etc